python/mxnet/ndarray/__init__.py

```diff
@@ -3,10 +3,10 @@
 from . import _internal
 from . import op
 from .op import CachedOp, invoke
-from .ndarray import NDArray, array, concatenate, _DTYPE_NP_TO_MX, _DTYPE_MX_TO_NP
-from .ndarray import empty, ones, add, arange, divide, equal, full, greater, greater_equal, imdecode
+from .ndarray import NDArray, concatenate, _DTYPE_NP_TO_MX, _DTYPE_MX_TO_NP
+from .ndarray import ones, add, arange, divide, equal, full, greater, greater_equal, imdecode
 from .ndarray import lesser, lesser_equal, maximum, minimum, moveaxis, multiply, negative, not_equal
 from .ndarray import onehot_encode, power, subtract, true_divide, waitall, _new_empty_handle
-from .ndarray_utils import load, save, zeros
-from .sparse_ndarray import _ndarray_cls
-from .sparse_ndarray import csr, row_sparse, SparseNDArray, todense, RowSparseNDArray, CSRNDArray
+from .ndarray_utils import load, save, zeros, empty, array
+from .sparse_ndarray import _ndarray_cls, todense
+from .sparse_ndarray import csr, row_sparse, BaseSparseNDArray, RowSparseNDArray, CSRNDArray
```

Comment:
`NDArrayBase` is not exposed to users. Since we also have `BaseModule` and `Module`, I thought `BaseSparseNDArray` could indicate the same thing. @nikrao what name do you think is better to use as the base class for CSRNDArray and RowSparseNDArray?